test/remote/gateways/remote_ipg_test.rb

```diff
@@ -101,10 +101,10 @@ def test_failed_purchase
   end
 
   def test_failed_purchase_with_passed_in_store_id
-    # passing in a bad store id results in a 401 unauthorized error
```

Comment:
Updated this because they do fail the transaction when the store_id doesn't match but it doesn't respond with a 401.